mantidproject/mslice
# Update cut/slice documentation details

**Description of work:**

Adds detailed documentation for how the `Slice` and `Cut` work (mathematically).

**To test:**

Please read the docs here:  
https://mducle.github.io/mslicedocs/math_ref.html
and build it and check that it makes sense and the equations and images render correctly.





No associated issue.
